modules/swagger-codegen/src/main/resources/htmlDocs2/sample_php.mustache

```diff
@@ -1,18 +1,18 @@
-
+<&#63;php
 require_once(__DIR__ . '/vendor/autoload.php');
 {{#hasAuthMethods}}{{#authMethods}}{{#isBasic}}
 // Configure HTTP basic authorization: {{{name}}}
-{{{invokerPackage}}}\Configuration::getDefaultConfiguration()->setUsername('YOUR_USERNAME');
-{{{invokerPackage}}}\Configuration::getDefaultConfiguration()->setPassword('YOUR_PASSWORD');{{/isBasic}}{{#isApiKey}}
+Swagger\Client\Configuration::getDefaultConfiguration()->setUsername('YOUR_USERNAME');
```

Comment:
Would it be better to use `{{{invokerPackage}}}` as PHP SDK owners will not always use the default (Swagger\Client) ?